CONTRIBUTING.md

```diff
@@ -56,10 +56,10 @@ before-mentioned sections).
 
 ## Common guidelines
 
-- Please make sure there is an issue associated with the work that you're doing.
+- Please make sure there is an issue associated with the work that you are doing.
 
-- If you're working on an issue, please comment that you are doing so to prevent
-  duplicate work by others also.
+- If you are working on an issue, please comment about the same to prevent
+  conflicts.
```

Comment:
Commenting that you're working is to prevent duplication of work, not to avoid conflicts. :smile: